fchost/fchost.go

```diff
@@ -32,7 +36,7 @@ func New() (*FilecoinHost, error) {
 		return nil, err
 	}
 
-	dht, err := dht.New(ctx, h)
+	dht, err := dht.New(ctx, h, dhtopts.Protocols("/lotus/kad/1.0.0"))
```

Comment:
Seems that now that libp2p hosts aren't running in a private network as before, they have their ProtocolID for DHT.
Took some time to figure out why DHT wasn't working...